client/scripts/controllers/chain/clover/main.ts

```diff
@@ -0,0 +1,20 @@
+import { ChainClass, NodeInfo } from 'models';
+import { IApp } from 'state';
+import Substrate from '../substrate/main';
+import spec from './spec';
+
+class Clover extends Substrate {
+  constructor(n: NodeInfo, app: IApp) {
+    super(n, app, ChainClass.Clover);
+    this.signaling.disable();
+    this.identities.disable();
+  }
+
+  public async initApi() {
+    await super.initApi({
+      'types': spec,
```

Comment:
I suggest modifying this to use the provided types package similarly to how you did in chain events. Then the `spec.ts` file can be deleted.